### PR TITLE
Bug 568711 Design model for grant acquisition

### PR DESCRIPTION
**Pull request by eparovyshnaya**
GrantAquisition class is identified, binds FeatureGrant, User and actual acquisition date.

Signed-off-by: eparovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.floating.ecore/model/floating.ecore [hunk 1]
```diff
@@ -87,4 +87,14 @@
     <eStructuralFeatures xsi:type="ecore:EReference" name="authentication" ordered="false"
         unique="false" lowerBound="1" eType="#//EvaluationInstructions" containment="true"/>
   </eClassifiers>
+  <eClassifiers xsi:type="ecore:EClass" name="GrantAcqisition">
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.floating.ecore/model/floating.ecore:90]: It is not referenced by other types. How do you plan to store it?

File: bundles/org.eclipse.passage.lic.floating.edit/src-gen/org/eclipse/passage/lic/floating/edit/providers/GrantAcqisitionItemProvider.java [hunk 2]
```diff
@@ -0,0 +1,219 @@
+/**
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.floating.edit/src-gen/org/eclipse/passage/lic/floating/edit/providers/GrantAcqisitionItemProvider.java:1]: header

File: bundles/org.eclipse.passage.lic.floating.model/src-gen/org/eclipse/passage/lic/floating/model/api/GrantAcqisition.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
+/**
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.floating.model/src-gen/org/eclipse/passage/lic/floating/model/api/GrantAcqisition.java:1]: header

File: bundles/org.eclipse.passage.lic.floating.model/src-gen/org/eclipse/passage/lic/floating/model/impl/GrantAcqisitionImpl.java [hunk 4]
```diff
@@ -0,0 +1,410 @@
+/**
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.floating.model/src-gen/org/eclipse/passage/lic/floating/model/impl/GrantAcqisitionImpl.java:1]: header

**Review by ruspl-afed — CHANGES_REQUESTED** [review]

File: bundles/org.eclipse.passage.lic.floating.ecore/model/floating.ecore (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.floating.ecore/model/floating.ecore:90]: It is going to be returned as a root element.

Comment (ruspl-afed) [reply bundles/org.eclipse.passage.lic.floating.ecore/model/floating.ecore:90]: ah ok, but we should ensure the corresponding loading code has no strong assumption regarding the file content

**Review by ruspl-afed — APPROVED** [review]

**Pull request merged**